app.py

```diff
@@ -13,16 +13,19 @@
 from distutils.dir_util import copy_tree
 from flask import Flask, render_template, request, send_from_directory, render_template_string
 from threading import Thread, Lock
-
+import sys
 # run at lower priority
 os.nice(20)
 
 #BOARDS = [ 'BeastF7', 'BeastH7' ]
 
 appdir = os.path.dirname(__file__)
 
-VEHICLES = [ 'Copter', 'Plane', 'Rover', 'Sub', 'Tracker' ]
+VEHICLES = [ 'Copter', 'Plane', 'Rover', 'Sub', 'Tracker', 'Blimp', 'Heli']
 default_vehicle = 'Copter'
+BRANCHES = ['upstream/master', 'upstream/Plane-4.2', 'upstream/Copter-4.2', 'upstream/Rover-4.2']
```

Comment:
upstream is just a convention. I suspect it might be easier to simply assume upstream and make the branches the actual branch name - master, Plane-4.2 etc

Reply:
The intent with that is that we can add branches from other remotes as well. For example, if you have a PR you want users to test, then you can just add eg 'andy/pr-test-branch' to that list (and the remote in the git repo itself) and then people can get the apj file from there rather than you having to build it individually for each person. 🙂

Reply:
I agree that we should make the names easier for regular users to understand.  I think the names should be, "latest", "Plane-4.2", "Copter-4.2" and "Rover-4.2".  So drop the "upstream" and rename "master" to "latest" which matches what we call it in the ground stations.

Reply:
I don't understand... That partially defeats the purpose of this PR as explained above...

And if I rename master to latest then git won't find it 'cause it'll look for a branch named "latest".